Foundation/NSMeasurement.swift

```diff
@@ -23,21 +23,94 @@ open class NSMeasurement : NSObject, NSCopying, NSSecureCoding {
         self.unit = unit
     }
     
-    open func canBeConverted(to unit: Unit) -> Bool { NSUnimplemented() }
+    open func canBeConverted(to otherUnit: Unit) -> Bool {
+        #if DEPLOYMENT_RUNTIME_OBJC //|| os(Linux)
+            return otherUnit.isKind(of: type(of: unit))
+        #else
+            // just check conversion
+            if let dimension = unit as? Dimension,
+               let otherDimension = otherUnit as? Dimension {
+                return true
```

Comment:
`dimension` and `otherDimension` are not used so could be replaced with `_` or perhaps rewrite as `unit is Dimension && otherUnit is Dimension`.

Reply:
I think I thought of that, then forgot to do it.